app/views/content/non-uk-teachers/why-train-to-teach-transcript.md

```diff
@@ -59,8 +59,6 @@ The value of gaining a transferable qualification through training to teach in E
 
```

Comment:
@laurahermionetennant92 we might need to delete the video or see if it's possible to edit the end captions? They currently show what you've deleted in the transcript so people will still be able to view the info about the IRP which isn't correct any more.

Reply:
Argh thank you – let me double check we do include a mention in the actual vid!

Reply:
@laurahermionetennant92 And it also says in the video you could be eligible for a bursary of up to £39K but everywhere else we say £30k?

Reply:
@laurahermionetennant92 I think you may still need to delete the transcript page too.